src/Context/Feed.tsx

```diff
@@ -0,0 +1,209 @@
+import {
+  Activity,
+  GetFeedOptions,
+  ReactionAPIResponse,
+  StreamClient,
+  ReactionAddOptions,
+  UR,
+  Reaction,
+  FeedAPIResponse,
+  ReactionAddChildOptions,
+  ReactionFilterAPIResponse,
+  ReactionFilterConditions,
+} from 'getstream';
+
+import { FeedManager } from './FeedManager';
+import { DefaultAT, DefaultUT, useStreamContext } from './StreamApp';
+
+export type FeedContextValue<
+  UT extends DefaultUT = DefaultUT,
+  AT extends DefaultAT = DefaultAT,
+  CT extends UR = UR,
+  RT extends UR = UR,
+  CRT extends UR = UR,
+  PT extends UR = UR
+> = {
+  activities: FeedManager<UT, AT, CT, RT, CRT, PT>['state']['activities'];
+  activityOrder: FeedManager<UT, AT, CT, RT, CRT, PT>['state']['activityOrder'];
+  feedGroup: string;
+  feedManager: FeedManager<UT, AT, CT, RT, CRT, PT>;
+  getActivityPath: FeedManager<UT, AT, CT, RT, CRT, PT>['getActivityPath'];
+  hasDoneRequest: boolean;
+  hasNextPage: boolean;
+  hasReverseNextPage: boolean;
+  loadNextPage: FeedManager<UT, AT, CT, RT, CRT, PT>['loadNextPage'];
+  loadNextReactions: FeedManager<UT, AT, CT, RT, CRT, PT>['loadNextReactions'];
+  loadReverseNextPage: FeedManager<UT, AT, CT, RT, CRT, PT>['loadReverseNextPage'];
+  onAddChildReaction: FeedManager<UT, AT, CT, RT, CRT, PT>['onAddChildReaction'];
+  onAddReaction: FeedManager<UT, AT, CT, RT, CRT, PT>['onAddReaction'];
+  onMarkAsRead: FeedManager<UT, AT, CT, RT, CRT, PT>['onMarkAsRead'];
+  onMarkAsSeen: FeedManager<UT, AT, CT, RT, CRT, PT>['onMarkAsSeen'];
+  onRemoveActivity: FeedManager<UT, AT, CT, RT, CRT, PT>['onRemoveActivity'];
+  onRemoveChildReaction: FeedManager<UT, AT, CT, RT, CRT, PT>['onRemoveChildReaction'];
+  onRemoveReaction: FeedManager<UT, AT, CT, RT, CRT, PT>['onRemoveReaction'];
+  onToggleChildReaction: FeedManager<UT, AT, CT, RT, CRT, PT>['onToggleChildReaction'];
+  onToggleReaction: FeedManager<UT, AT, CT, RT, CRT, PT>['onToggleReaction'];
+  realtimeAdds: FeedManager<UT, AT, CT, RT, CRT, PT>['state']['realtimeAdds'];
+  realtimeDeletes: FeedManager<UT, AT, CT, RT, CRT, PT>['state']['realtimeDeletes'];
+  refresh: FeedManager<UT, AT, CT, RT, CRT, PT>['refresh'];
+  refreshing: boolean;
+  refreshUnreadUnseen: FeedManager<UT, AT, CT, RT, CRT, PT>['refreshUnreadUnseen'];
+  unread: number;
+  unseen: number;
+  userId?: string;
+};
+
+export type FeedProps<
+  UT extends DefaultUT = DefaultUT,
+  AT extends DefaultAT = DefaultAT,
+  CT extends UR = UR,
+  RT extends UR = UR,
+  CRT extends UR = UR,
+  PT extends UR = UR
+> = {
+  /** The feed group part of the feed */
+  feedGroup: string;
+  /** The location that should be used for analytics when liking in the feed,
+   * this is only useful when you have analytics enabled for your app. */
+  analyticsLocation?: string;
+  /** Override activity delete request */
+  /* Components to display in the feed */
+  children?: React.ReactNode;
+  doActivityDeleteRequest?: (id: string) => Promise<void | unknown>;
```

Comment:
Might be a good idea to give this function type a name and use it with `doChildReactionDeleteRequest` too.